manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -72,6 +72,7 @@ type DataContext struct {
 	IFdetails InterfaceDetails `json:"ifDetails"`
 
 	// Recurrence indicates how often the copy should take place or if streamed input is needed
+	// Default is none
```

Comment:
unless you add dots this looks bad in the generated docs:

> description: Recurrence indicates how often the copy should take place or if streamed input is needed Default is none

Reply:
fixed